CorsixTH/Lua/world.lua

```diff
@@ -1227,7 +1221,8 @@ function World:winGame(player_no)
     local bonus_rate = math.random(4,9)
     local with_bonus = self.ui.hospital.cheated and 0 or (self.ui.hospital.player_salary * bonus_rate) / 100
     self.ui.hospital.salary_offer = math.floor(self.ui.hospital.player_salary + with_bonus)
-    if type(self.map.level_number) == "number" or self.campaign_info then
+    local isCampaign = type(self.map.level_number) == "number" or self.campaign_info
+    if isCampaign then
```

Comment:
I'd revert this. Or make a new function for it before `World:getCampaignData()`

```lua
--! Returns whether the level being played is part of a campaign or not
function World:isCampaign()
  return type(self.map.level_number) == "number" or self.campaign_info
end
```

Then
```suggestion
    if self:isCampaign() then

Reply:
> Or make a new function for it before `World:getCampaignData()`

Thanks. Applied.